Common/Data/Auxiliary/LocalDiskFactorFileProvider.cs

```diff
@@ -71,7 +72,7 @@ public FactorFile Get(Symbol symbol)
                 return GetFactorFile(symbol, symbol.Value, market);
             }
 
-            var mapFile = mapFileResolver.ResolveMapFile(symbol.Value, symbol.ID.Date);
+            var mapFile = mapFileResolver.ResolveMapFile(symbol.Value, DateTime.Today);
```

Comment:
Consider goocv before it was goog in a backtest. We would ask the map for resolver to find a goocv (symbol.Value) active today, but they're isn't one. Maybe what it should be doing is symbol.ID.Symbol and symbol.ID.Date. This would say give me the mail file based on the symbols first ticker/date combination.